README.md

```diff
@@ -1 +1,71 @@
 # Boundary: Nomad Client-based Boundary Workers
+
+This Terraform Module provisions resources for a Nomad Client-based Boundary worker deployment.
+
+## Table of Contents
+
+<!-- TOC -->
+* [Boundary: Nomad Client-based Boundary Workers](#boundary-nomad-client-based-boundary-workers)
+  * [Table of Contents](#table-of-contents)
+  * [Requirements](#requirements)
+  * [Usage](#usage)
+    * [Inputs](#inputs)
+    * [Outputs](#outputs)
+  * [Author Information](#author-information)
+  * [License](#license)
+<!-- TOC -->
+
+## Requirements
+
+* HashiCorp HCP Boundary `0.14.0` or newer
+* HashiCorp Nomad `1.5.0` or newer
+* HashiCorp Terraform `1.5.0` or newer.
+
+## Usage
+
+> **Warning**
+> Interacting with sensitive data from Terraform causes any secrets that you read and write to be persisted in Terraform's state file and in any generated plan files.
+
+For examples, see the [./examples](https://github.com/ksatirli/terraform-vault-kv-v2/tree/main/examples/) directory.
+
+<!-- BEGIN_TF_DOCS -->
+### Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| boundary_account_password | Account Password of the Boundary User for Nomad Clients. | `string` | n/a | yes |
+| boundary_auth_method_id | Auth Method Id of the Boundary Controller. | `string` | n/a | yes |
+| boundary_auth_method_login_name | Auth Method Login Name of the Boundary Controller. | `string` | n/a | yes |
+| boundary_auth_method_password | Auth Method Password of the Boundary Controller. | `string` | n/a | yes |
+| hcp_boundary_cluster_id | ID of the HCP Boundary Controller. | `string` | n/a | yes |
+| nomad_variable_path | Path of the Nomad Variable. | `string` | n/a | yes |
```

Comment:
A description of what the Nomad Var is used for could be useful

Reply:
I've added this in the `Notes` section, and the code.